data/ssystem_minor.ini

```diff
@@ -947,7 +1048,8 @@ coord_func                     = comet_orbit
 dust_brightnessfactor          = 1.5
 dust_lengthfactor              = 0.4
 dust_widthfactor               = 1.5
-name                           = 2I/Borisov [C/2019 Q4 (Borisov)]
+name                           = 2I/Borisov
```

Comment:
Also here:

name=Borisov
iau_designation=2I/2019 Q4
discovery_code=C/2019 Q4

Reply:
The `discovery_code`, `date_code` and `perihelion_code` supporting by comets only. Probably I should add support of them for all minor bodies (at least for `A/` and `I/` objects).

This object hasn't proper name, so, `name` should follow IAU (provisional or standard) designation.